packages/styles/panel.css

```diff
@@ -0,0 +1,58 @@
+:root {
+  --panel-border-color: var(--gray-40);
+  --panel-header-background-color: var(--gray-20);
+  --panel-header-color: var(--gray-90);
+  --panel-content-background-color: var(--white);
+  --panel-content-color: var(--gray-60);
+}
+
+.Panel {
+  border: 1px solid var(--panel-border-color);
+}
+
+.Panel__Header {
+  background-color: var(--panel-header-background-color);
+  color: var(--panel-header-color);
+  font-size: var(--text-size-normal);
+  display: flex;
+  height: 35px;
+  border-bottom: 1px solid var(--panel-border-color);
+}
+
+.Panel__Header-title {
+  flex-grow: 1;
+  display: flex;
+  line-height: var(--space-gutter);
+  margin-left: var(--space-small);
+}
+
+.Panel__Header-actions {
+  flex-grow: 1;
+  text-align: right;
+  line-height: var(--space-gutter);
+  margin-right: var(--space-smallest);
+}
+
+.Panel__Header-actions button {
+  margin-bottom: var(--space-smallest);
+}
+
```

Comment:
```suggestion
```

If we set vertical alignment with `align-items` above, this shouldn't be needed anymore.